modules/spring-config/src/main/java/org/mule/runtime/config/internal/dsl/model/DefaultArtifactDeclarationXmlSerializer.java

```diff
@@ -83,43 +85,43 @@ private String serializeArtifact(ArtifactDeclaration artifact) {
     checkArgument(artifact != null, "The artifact to serialize cannot be null");
 
     try {
-      Document doc = createAppDocument(artifact);
+      final Document doc = createAppDocument(artifact);
 
-      XmlDslElementModelConverter toXmlConverter = XmlDslElementModelConverter.getDefault(doc);
-      DslElementModelFactory modelResolver = DslElementModelFactory.getDefault(context);
+      final XmlDslElementModelConverter toXmlConverter = XmlDslElementModelConverter.getDefault(doc);
+      final DslElementModelFactory modelResolver = DslElementModelFactory.getDefault(context);
 
       artifact.getGlobalElements()
           .forEach(declaration -> appendChildElement(toXmlConverter, doc.getDocumentElement(),
                                                      modelResolver, (ElementDeclaration) declaration));
 
-      List<String> cDataElements = getCDataElements(doc.getDocumentElement());
+      final List<String> cDataElements = getCDataElements(doc.getDocumentElement());
 
       // write the content into xml file
-      TransformerFactory transformerFactory = XMLSecureFactories.createDefault().getTransformerFactory();
+      final TransformerFactory transformerFactory = XMLSecureFactories.createDefault().getTransformerFactory();
 
-      Transformer transformer = transformerFactory.newTransformer();
+      final Transformer transformer = transformerFactory.newTransformer();
       transformer.setOutputProperty(OutputKeys.INDENT, "yes");
       transformer.setOutputProperty(OutputKeys.CDATA_SECTION_ELEMENTS, join(cDataElements, " "));
       transformer.setOutputProperty("{http://xml.apache.org/xslt}indent-amount", "4");
 
-      DOMSource source = new DOMSource(doc);
-      StringWriter writer = new StringWriter();
+      final DOMSource source = new DOMSource(doc);
+      final StringWriter writer = new StringWriter();
       transformer.transform(source, new StreamResult(writer));
       return writer.getBuffer().toString();
 
-    } catch (Exception e) {
+    } catch (final Exception e) {
       throw new MuleRuntimeException(createStaticMessage("Failed to serialize the declaration for the artifact ["
           + artifact.getName() + "]: " + e.getMessage()), e);
     }
   }
 
   private Document createAppDocument(ArtifactDeclaration artifact) throws ParserConfigurationException {
-    DocumentBuilderFactory factory = DocumentBuilderFactory.newInstance();
+    final DocumentBuilderFactory factory = DocumentBuilderFactory.newInstance();
```

Comment:
Should we be setting the features on this one too?

Reply:
yep, good catch